ocs_ci/templates/workloads/hsbench/hsbench_obj.yaml

```diff
@@ -0,0 +1,15 @@
+uid: "johndoe"
+display_name: '"John Doe"'
+email: "john@example.com"
+access_key: 'UPUS6I40TCFN34S6SJMJ'
```

Comment:
Hope we are handling to get the access_key and secret_key from ceph and accordingly it will be updated in this yaml.

Reply:
I think it's handled in line number 96/97